qrify/qrify

```diff
@@ -80,34 +80,25 @@ update()
 
 makeqr()
 {
-  input=$(echo ${input} | sed s/" "/%20/g ) ## replace all spaces in the sentence with HTML-encoded space %20
-  httpGet qrenco.de/${input} ## get a response for the qrcode
+  input=$(echo $input | sed s/" "/%20/g ) ## replace all spaces in the sentence with HTML-encoded space %20
+  httpGet qrenco.de/$input ## get a response for the qrcode
 }
 
 # redirects the image obtained from the goqr api into a png file
 function makeQRFile {
-	input=$(echo "${input}" | sed -e s/" "/%20/g -e s/'\\n'/%0A/g ) ##same as in the makeqr function
-	httpGet "api.qrserver.com/v1/create-qr-code/?size=150x150&data=${input}" > "$fileName"
+	input=$(echo "$input" | sed -e s/" "/%20/g -e s/'\\n'/%0A/g ) ##same as in the makeqr function
+	httpGet "api.qrserver.com/v1/create-qr-code/?size=150x150&data=$input" > $fileName.png
```

Comment:
Just asking, but if the user enters the command as `$ qrify -f image.png google.com`, will this not store the qr code in image.png.png?

Reply:
It will, I do not have the full time to test but on Unix which uses bash 3 something you did something in the function where you checked for the file extension that Im pretty sure wouldn't have worked. This can be fixed though just by grepping to see if the user has entered .png as the last part and if not append it in between lines 89 and 90.

Reply:
Oh all right. Thanks for clearing it up!

Reply:
No problem! I don't get to use all those cool bash things because unix is so far behind, it has caused many roadblocks in my quest to be as cross-platform as possible

Reply:
Oh really? What are you using as of now?

Reply:
I mostly use Elementary OS for day to day stuff and programming, I use Windows for some of my Engineering programs when I started this project i was using OSX instead of Windows so I decided to make this project work for all 3. I even try to support as many types of Linux as possible although it doesn't always work out

Reply:
Oh cool! Although supporting various platforms does become difficult. One of my scripts (not in qrify) essentially fails to run on OSX because I used a command available only in GNU/Linux bash, not BSD.